src/adapter/tests.rs

```diff
@@ -1657,3 +1657,131 @@ fn unions() {
 
     similar_asserts::assert_eq!(expected_results, results);
 }
+
+#[test]
+fn enum_discriminants() {
+    let path = "./localdata/test_data/enum_discriminants/rustdoc.json";
+    let content = std::fs::read_to_string(path)
+        .with_context(|| format!("Could not load {path} file, did you forget to run ./scripts/regenerate_test_rustdocs.sh ?"))
+        .expect("failed to load rustdoc");
+
+    let crate_ = serde_json::from_str(&content).expect("failed to parse rustdoc");
+    let indexed_crate = IndexedCrate::new(&crate_);
+    let adapter = RustdocAdapter::new(&indexed_crate, None);
+
+    let query = r#"
+{
+    Crate {
+        item {
+            ... on Enum {
+                name @output
+                variant {
+                    discriminant {
```

Comment:
Would you mind outputting the variant names here, so we're positive we're correctly matching up variants and discriminants?

If at any point we happened to introduce a bug that makes that not be the case, finding it would be a horrifying debugging experience to go through.